AudioLinkWebProject/.gitignore

```diff
@@ -0,0 +1,75 @@
+# This .gitignore file should be placed at the root of your Unity project directory
```

Comment:
seperate gitignore? is this really necessary

Reply:
bro https://github.com/llealloo/audiolink/blob/master/AudioLinkUnityProject%2F.gitignore unity creates this when you make a new project

Reply:
TIL